src/Core__Ordering.res

```diff
@@ -0,0 +1,13 @@
+type t = float
+
+let less = -1.
+let equal = 0.
+let greater = 1.
```

Comment:
I think these should have `@inline`.

Reply:
Good idea! Done in https://github.com/rescript-association/rescript-core/pull/149/commits/65028a7c41dd2d773d6f5efdba7cb0d755f9f185